doc/source/tutorial/tutorial_IO.ipyml

```diff
@@ -9,9 +9,19 @@ cells:
 
 
 - code: |
-    from larray import *
+    # run this cell to avoid annoying warnings
+    import warnings
+    warnings.filterwarnings("ignore")
```

Comment:
it would be better to be more specific because we could silence our own warnings with this, which is bad